test/addons-abi/7_factory_wrap/myobject.h

```diff
@@ -13,10 +13,12 @@ class MyObject {
   MyObject();
   ~MyObject();
 
-  static napi_persistent constructor;
+  static napi_ref constructor;
   static void New(napi_env env, napi_callback_info info);
   static void PlusOne(napi_env env, napi_callback_info info);
   double counter_;
+  napi_env env_;
```

Comment:
Just an api ergonomics question- can we store the environment in the wrapped object so that people don't need to store a reference to the environment for use later?